src/sentry/tasks/clear_expired_resolutions.py

```diff
@@ -23,7 +23,7 @@ def clear_expired_resolutions(release_id):
     resolution_list = list(
         GroupResolution.objects.filter(
             Q(type=GroupResolution.Type.in_next_release) | Q(type__isnull=True),
-            release__projects=release.projects.all(),
+            release__projects__in=[p.id for p in release.projects.all()],
```

Comment:
Should this be clearing expired resolutions for all groups across all projects?

If so, we can just remove the usage of ReleaseProject entirely?

If not, can we just query for `GroupResolution.project_id = project_id`?

Reply:
No, it should be clearing expired resolutions for projects that are associated with the current release, and there can be multiple related projects. There's no `GroupResolution.project_id`, if you see one in the database it's deprecated, since it doesn't exist on the model and so we're not writing to it.

Reply:
Damn it I read it as `project_id` and not as `group_id`. Really glad we don't write this, it's not like a Group can transfer projects. 🙃